compiler/src/iree/compiler/Codegen/LLVMGPU/ROCDLPrefetching.cpp

```diff
@@ -36,9 +38,13 @@ static void prefetchSharedMemory(FunctionOpInterface funcOp) {
 
 struct ROCDLPrefetchSharedMemoryPass final
     : impl::ROCDLPrefetchSharedMemoryPassBase<ROCDLPrefetchSharedMemoryPass> {
+  using Base =
+      impl::ROCDLPrefetchSharedMemoryPassBase<ROCDLPrefetchSharedMemoryPass>;
```

Comment:
You shouldn't need this -- tablegen should generate this automatically, doesn't it?